en/docs/references/pricing.md

```diff
@@ -0,0 +1,229 @@
+
+#### Pay As You Go (PAYG) 
+Pay As You Go is a flexible usage-based payment plan. While the first 5,000 steps are free per month, Choreo charges you $0.005 for each additional step based on your usage per month. For example, for 1,000 steps Choreo charges you $5. 
+
+Let’s assume you have used a total of 12,500 [steps](#what-is-a-step) for a particular month, your monthly usage for that month under the PAYG plan will be:
+
+    Chargeable steps usage: 12,500 - 5,000 = 7,500
+    The total charge for the usage: 7,500 * $0.005 ($5 for every 1,000 steps) = $37.50
+
+#### Enterprise 
+The Enterprise plan is suitable for large-scale usage of Choreo. You can access volume discounts and committed-use discounts with this plan. [Contact sales](https://wso2.com/contact/?ref=choreo) to get a discounted rate.
+
+### What is a step?
+A step (1 step) is a maximum of 500ms compute time on a container having 350MB of memory and 0.5 vCPU. Any additional compute time is counted as an additional step.  
+An incoming event or message to Choreo or an outgoing API call from Choreo is a minimum of 1 step.
+
+
+## Pricing examples
+
+### Example 1: REST API 
+
+An organization has a flight booking system on Choreo. This system involves several APIs and databases. The scenario and the APIs being used in the system are described below.
+
+#### Scenario
+
+To make a booking through the web application, a client first searches for available flights and then proceeds to make the booking. The client application (web application) upon receiving the request makes a call to the [Search API](#searchAPI) to search for available flights. When the client receives the search results the client proceeds to make a reservation using the [Reservations API](#reservationsAPI).
+
+The following two diagrams illustrate the interactions in the system while searching and reserving a flight.
+<br/>
+
+#### Glossary
+- <a class="pricing" id="searchAPI"> Search API</a>: An external-facing API in Choreo that searches for the availability of flights and seats given the destinations.
+- <a class="pricing" id="bookingsAPI">Bookings API</a>: An internal API in Choreo that updates the Bookings database when a booking for a certain flight is made.
+- <a class="pricing" id="reservationsAPI">Reservations API</a>: An external-facing API in Choreo that is used by client applications (web apps, mobile apps) to make flight bookings.
+- <a class="pricing" id="twillioAPI">Twilio API</a>: A third-party API that is used by the Reservations API in Choreo to send SMS notifications to customers when a booking has been made, canceled, or updated.
+- <a class="pricing" id="executionTime">Execution time</a>: The execution time taken for a process is the time taken between  sending a request and recieving a response.
```

Comment:
There is an extra space after "between" and "received" is misspelt.

Reply:
done